lib/application.ts

```diff
@@ -78,7 +79,7 @@ export class SNApplication {
 
   public environment: Environment
   public platform: Platform
-  public namespace: string
+  private fixedNamespace?: string
```

Comment:
Seems like this should be called `namespaceIdentifier` instead? That's how it appears to be used. This should also have a comment in the application constructor as to why/when you would pass an identifier for a namespace.

Reply:
(On line 126)